api/jobimp/v1/jobimp.proto

```diff
@@ -0,0 +1,41 @@
+syntax = "proto3";
```

Comment:
Should these protos go under the existing [`omeapi` directory](https://github.com/openmarketplaceengine/openmarketplaceengine/tree/main/omeapi)?

Reply:
`omeapi` was already configured to generate protobuf files into `internal` folder